examples/creating_a_simulation_library/simulating_diffraction_patterns.py

```diff
@@ -0,0 +1,93 @@
+"""
+=====================================================
+Simulating One Diffraction Pattern for a Single Phase
```

Comment:
Rename to something like *Simple diffraction pattern simulations*, if that is the intent of the example? Subsections can then be shortened to something like:
* One pattern for one phase
* Multiple patterns for one phase
* Multiple patterns for multiple phases
* Plotting a "Real" Diffraction Pattern